src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -531,6 +541,7 @@ VirtualRepeatController.prototype.getItemSize = function() {
  * @private
  */
 VirtualRepeatController.prototype.virtualRepeatUpdate_ = function(items, oldItems) {
+  this.isVirtualRepeatUpdating_ = true;
```

Comment:
Please add a blank line after this and one before the related line at the bottom of this method.